manifests/repo/yum.pp

```diff
@@ -8,12 +8,16 @@
     if $sensuclassic::repo_source {
       $url = $sensuclassic::repo_source
     } else {
+      $major = $facts.dig('os', 'release', 'major')
+      $family = $facts.dig('os', 'family')
       if $::operatingsystem == 'Amazon' {
-        if $facts['os']['release']['major'] =~ /^201\d$/ {
+        if $major =~ /^201\d$/ {
```

Comment:
regex's should have comments above explaining in plain wording what they are meant to match

Reply:
Comment added